### ISSUE TITLE
Could not find cn.aigestudio.wheelpicker:WheelPicker:1.0.3.

### ISSUE THREAD

**Issue by SandeshVakale** [devaige/WheelPicker#168]
Hello,

I am trying to use wheel Picker in my react native app.
I am using a react-native-wheel-picker library that uses WheelPicker from AigeStudio for android.

But I am getting the following error.

````
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find cn.aigestudio.wheelpicker:WheelPicker:1.0.3.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/cn/aigestudio/wheelpicker/WheelPicker/1.0.3/WheelPicker-1.0.3.pom
       - file:/Users/sandeshvakale/.m2/repository/cn/aigestudio/wheelpicker/WheelPicker/1.0.3/WheelPicker-1.0.3.pom
       - file:/Users/sandeshvakale/Desktop/Projects/Maaf/node_modules/react-native/android/cn/aigestudio/wheelpicker/WheelPicker/1.0.3/WheelPicker-1.0.3.pom
       - file:/Users/sandeshvakale/Desktop/Projects/Maaf/node_modules/jsc-android/dist/cn/aigestudio/wheelpicker/WheelPicker/1.0.3/WheelPicker-1.0.3.pom
       - https://dl.google.com/dl/android/maven2/cn/aigestudio/wheelpicker/WheelPicker/1.0.3/WheelPicker-1.0.3.pom
       - https://www.jitpack.io/cn/aigestudio/wheelpicker/WheelPicker/1.0.3/WheelPicker-1.0.3.pom
     Required by:
         project :app > project :gregfrench_react-native-wheel-picker
````

I also forced the library to use the latest 1.1.3 version by making changes in the library Gradle file.

But I get the same error again.

Is anyone facing the same issue?

Thanks in advance for your help.

**Comment by akhilsanker** [comment]
@SandeshVakale ,

Have you got any solution. I am facing the same in react-native 0.66.2.

Thanks

**Comment by SandeshVakale** [comment]
Nope, I changed design for android and used picker from rn community. But you can also use picker from react native ui lib by wix. They have one nice picker.

**Comment by luiscesjr** [comment]
I started getting this same error, and like many other times you start to wonder WTH is going on, and forget simple things. In my case specifically, I removed the jcenter() repo, so maybe that helps whoever else gets this error.

Better start looking into a different option if it's on that repo only...

**Comment by zabojad** [comment]
Add this to your `build.graddle` file and it will work:

    allprojects {
      repositories {
        // ...
        jcenter() {
            content {
                includeModule("cn.aigestudio.wheelpicker", "WheelPicker")
            }
        }

**Comment by swatiredhu17** [comment]
@zabojad This solves above issue but JCenter shutting down, so trying to remove all jcenter dependancy from project. Doesn't it affect that? 

> Add this to your `build.graddle` file and it will work:
> 
> ```
> allprojects {
>   repositories {
>     // ...
>     jcenter() {
>         content {
>             includeModule("cn.aigestudio.wheelpicker", "WheelPicker")
>         }
>     }
> ```

**Comment by zabojad** [comment]
@swatiredhu17 the snippet I've shared allows to use jcenter only for the wheelpicker.

**Comment by raphael-dario** [comment]
so it actually doesn't solve anything if JCenter is down. the only viable option I see in the long term is to host it on another maven repo, maybe Jitpack.io. you guys need to Draft a New Release here on Github with a version tag, and then it will get picked up and built by Jitpack here:
https://jitpack.io/#AigeStudio/WheelPicker

**Comment by batuhanbag** [comment]
> so it actually doesn't solve anything if JCenter is down. the only viable option I see in the long term is to host it on another maven repo, maybe Jitpack.io. you guys need to Draft a New Release here on Github with a version tag, and then it will get picked up and built by Jitpack here: https://jitpack.io/#AigeStudio/WheelPicker

yeah I think so, this is just temporary solution

**Comment by batuhanbag** [comment]
> 

You're welcome

**Comment by Hung6129** [comment]
any updates guys... ?

**Comment by dsandmark** [comment]
There's a solution here: https://github.com/devaige/WheelPicker/issues/179#issuecomment-2399233032
